src/nyc_trees/apps/event/views.py

```diff
@@ -23,8 +23,9 @@
 
 
 def event_dashboard(request):
-    # TODO: implement
-    pass
+    base_url = reverse('group_detail',
+                       kwargs={'group_slug': request.group.slug})
+    return HttpResponseRedirect(base_url + '#group-events')
```

Comment:
Correct, and also I am going to update this PR to rip out event dashboard entirely, because it has been replaced by the event tab of the group admin.